packages/core/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@geostreams/core",
+  "version": "3.6.1",
+  "main": "src/index.jsx",
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "NCSA",
+  "license": "ISC",
+  "scripts": {
+    "start": "webpack serve --open --config webpack.dev.js",
+    "lint": "eslint -c .eslintrc --ext .js --ext .jsx --ignore-path .eslintignore ./",
+    "lint:fix": "eslint -c .eslintrc --ext .js --ext .jsx --ignore-path .eslintignore --fix ./",
+    "flow": "flow"
+  },
+  "dependencies": {
+    "clsx": "^1.1.1",
+    "d3": "~5.16.0",
+    "ol": "~6.5.0",
+    "ol-ext": "~3.1.14",
+    "ol-layerswitcher": "~3.8.0"
+  },
+  "peerDependencies": {
```

Comment:
Do we need to mention these dependencies here? They are already in the root `package.json`.

Reply:
I removed them from the root package.json because root dependencies are treated as common dependencies for all packages, which means they will be added to the package.json of all packages as well when publishing. Since the referenced packages are only used by core and geostreaming, adding them to the root package.json will cause issues in core__old and unnecessary dependency requirement for eslint-config.

Reply:
Make sense.